### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the listitems in the table in "Browse Repositories dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1475]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. In "settings" dialog , tab to and select plugin tree item.
4. Tab to and select "Browse Repositories" button .
5. In "Browse Repositories " dialog ,.Tab to list items in the table and observe the Jaws and NVDA behaviour

**Actual Result:**
Jaws and NVDA are not announcing the name of the list items present in the table.

**Jaws Behaviour:**
table

**NVDA Behaviour:**
table
selected row 3 column 1
tab
selected row 4 column 1

**Expected Result:**
Jaws and Narrator should announce the name of the list items present in the table

Jaws and NVDA should announce as
360FireLinePlugin 
CODETOOLS  19,536 , 5 rating ,
one day ago
selected row 3 column 1

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![browse repositories-listitems](https://user-images.githubusercontent.com/22956130/36104101-1615c326-1037-11e8-803b-08a67b565bed.jpg)
![1f3dd71c](https://user-images.githubusercontent.com/22956130/36104107-19488402-1037-11e8-8d72-887506d29374.jpg)
![browse repositories-list items-explorer](https://user-images.githubusercontent.com/22956130/36104108-1bdf85b2-1037-11e8-83d7-a80100a4d0cf.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749916
